src/target/cortexm.c

```diff
@@ -698,7 +698,7 @@ bool cortexm_probe(adiv5_access_port_s *ap)
 	case JEP106_MANUFACTURER_RENESAS:
 		PROBE(renesas_probe);
 		break;
-	case JEP106_MANUFACTURER_ARM_CHINA:
+	case JEP106_MANUFACTURER_ERRATA_ARM_CHINA:
```

Comment:
Shouldnt this code be replaced when found during the scan, like the other errata codes?

Reply:
ok, replaced.